src/prefect/server/schemas/graph.py

```diff
@@ -31,17 +37,17 @@ class Node(PrefectBaseModel):
     id: UUID
     label: str
     state_type: StateType
-    start_time: datetime
-    end_time: Optional[datetime]
+    start_time: DateTime
+    end_time: Optional[DateTime]
     parents: List[Edge]
     children: List[Edge]
     encapsulating: List[Edge]
     artifacts: List[GraphArtifact]
 
 
 class Graph(PrefectBaseModel):
-    start_time: datetime
-    end_time: Optional[datetime]
+    start_time: Optional[DateTime]
```

Comment:
`Graph.start_time` is `Optional[]` because nothing ensures that it isn't nullable.

The query from which this value is drawn is `SELECT COALESCE(start_time, expected_start_time), end_time FROM flow_run WHERE id = :flow_run_id`, and both `start_time` and `expected_start_time` are nullable.